src/app/submissions/submissions.service.ts

```diff
@@ -52,6 +52,23 @@ export class SubmissionsService {
     });
   }
 
+  updateSubmissonNotifcation () {
+    const data = {
+      'message': this.userService.get().name + ' has complete the survey ' + this.submission.parent.name,
+      'link': '/surveys',
+      'type': 'survey',
+      'priority': 1,
+      'status': 'unread',
+      'time': this.couchService.datePlaceholder
+    };
+    // todo fork join two notification send request
+    forkJoin([
```

Comment:
this will duplicate the notification if same user create and send survey. so fix this